libraries/rig-package/src/RigConfig.ts

```diff
@@ -337,6 +337,56 @@ export class RigConfig {
     return this._resolvedProfileFolder;
   }
 
+  /**
+   * This lookup first checks for the specified relative path under `projectFolderPath`; if it does
+   * not exist there, then it checks in the resolved rig profile folder.  If the file is found,
+   * its absolute path is returned. Otherwise, `undefined` is returned.
+   *
+   * @remarks
+   * For example, suppose the rig profile is:
+   *
+   * `/path/to/your-project/node_modules/example-rig/profiles/example-profile`
+   *
+   * And suppose `configFileRelativePath` is `folder/file.json`. Then the following locations will be checked:
+   *
+   * `/path/to/your-project/folder/file.json`
+   *
+   * `/path/to/your-project/node_modules/example-rig/profiles/example-profile/folder/file.json`
+   */
+  public tryResolveConfigFilePath(configFileRelativePath: string): string | undefined {
+    const localPath: string = path.join(this.projectFolderPath, configFileRelativePath);
```

Comment:
```suggestion
    const localPath: string = path.resolve(this.projectFolderPath, configFileRelativePath);
```

Reply:
@iclanton Hmm... it seems wrong that `projectFolderPath` is not an absolute path.

I've fixed that in this commit: https://github.com/microsoft/rushstack/pull/2232/commits/61c93c45901427a9d3c7335e9448413b571233f0